e2e-testing/cypress/functions/PostFunctions.js

```diff
@@ -0,0 +1,61 @@
+/// <reference types="Cypress" />
+
+import PostLocators from '../locators/PostLocators';
+
+class PostFunctions {
+  constructor() {
+    this.postTitle = `Automated Title Response-${Math.floor(Math.random() * 900) + 100}`;
+    this.postDescription = 'Automated Description Response';
+  }
+  open_post_creation_form() {
+    cy.get('.sidebar__add-post-btn .submit-post-button .mzima-button--primary').click();
+    cy.get(PostLocators.srvyItemBtn).click();
+  }
+
+  fill_required_form_fields() {
+    cy.contains('mat-label', 'Title')
+      .siblings('mat-form-field')
+      .find('input.mat-input-element')
+      .type(this.postTitle);
+    cy.get(PostLocators.descField).type(this.postDescription);
+    cy.get(PostLocators.lngTextField).type('This is an automated long text response');
+    cy.contains('mat-label', 'Number (Decimal) Field')
+      .siblings('mat-form-field')
+      .find('input.mat-input-element')
+      .type(99.9);
+    cy.contains('mat-label', 'Number (Integer) Field')
+      .siblings('mat-form-field')
+      .find('input.mat-input-element')
+      .type(100);
+    // click on date field to open pop up
+    cy.contains('mat-label', 'Date & Time Field')
+      .siblings('mat-form-field')
+      .find('input.mat-input-element')
+      .click();
+    cy.get('.mat-button.mat-stroked-button').click(); // click on the checkmark button to select today's date
+    cy.get(PostLocators.selectFieldBtn).click();
+    cy.get(PostLocators.selectFieldOption1).click();
+    cy.get(PostLocators.radioFieldOption2).click();
+    cy.get(PostLocators.checkboxFieldOption3).click();
+    cy.contains('mat-label', 'Short text Field (last)')
```

Comment:
The short text field has a unique selector `cy.get('[data-qa="short-text field"]')`
Please use it.

Reply:
@AmTryingMyBest I tried to use the unique selector, but it doesn't exist on short text field which is why I decided to go this route. When I inspect the component, data-qa for that field is null

<img width="406" alt="Screenshot 2023-10-16 at 14 17 08" src="https://github.com/ushahidi/platform-client-mzima/assets/64274826/1776fe15-d123-4acf-9fda-222302001aa2">

Reply:
Hmm. Add it as I shared, I'll check the run against the test environment and let you know what happens.

Reply:
alright. I just did that 👍